sources/api/migration/migrator/src/main.rs

```diff
@@ -82,11 +124,65 @@ fn run() -> Result<()> {
             process::exit(0);
         });
 
-    let migrations = find_migrations(
-        &args.migration_directories,
-        &current_version,
-        &args.migrate_to_version,
-    )?;
+    // DEPRECATED CODE BEGIN ///////////////////////////////////////////////////////////////////////
+    // check if the `from_version` supports signed migrations. if not, run the 'old'
+    // unsigned migrations code and return.
+    if !are_migrations_signed(&current_version) {
```

Comment:
@tjkirch left a comment here that, inexplicably, GitHub allowed me to delete when I was, in fact, trying to delete my own response. Fortunately I have both pasted to a text file:

@tjkirch wrote
> I'd rather remove the code entirely when we don't want to support unsigned migrations. I've found it's usually not worth having the branches, because they can go wrong on their own, you can forget them, etc.

And my response:
The branch is required for correctness. I don't understand what you're suggesting. If we don't check the version and choose a branch then we are either supporting signed or unsigned, but not both.

Reply:
Sorry, I guess that was a more general comment since I was looking at the branching/comments in multiple places regarding deprecation, and it was throwing me off overall.  Yeah, obviously we need to pick here, but we have more pertinent/correct information like the repo / signed migrations being available, right?  If that's too complicated I don't mind this.

Reply:
We will stick with the current implementation. The main risk is that we need to change the value of the version constant if our release plans change. Lets both call it out if that happens! Thanks.